go/vt/mysqlctl/capabilityset.go

```diff
@@ -46,34 +46,10 @@ func (c *capabilitySet) hasMySQLUpgradeInServer() bool {
 func (c *capabilitySet) hasInitializeInServer() bool {
 	return c.isMySQLLike() && c.version.atLeast(serverVersion{Major: 5, Minor: 7, Patch: 0})
 }
-func (c *capabilitySet) hasMySQLxEnabledByDefault() bool {
```

Comment:
do we expect these to be useful later on? Or we want them here for completeness? It is possible to ask staticcheck to ignore them.

Reply:
Yes, I would like to use them in the future. But we can remove them for now, they are easy enough to re-add.